swift_browser_ui_frontend/src/common/store.js

```diff
@@ -2,8 +2,12 @@
 import Vue from "vue";
 import Vuex from "vuex";
 
-import { recursivePruneCache } from "@/common/conv";
+// import { recursivePruneCache } from "@/common/conv";
```

Comment:
is this a forgotten comment?

Reply:
Yup, from when I was thinking if I should re-implement the caching anyway

I'll remove those

Reply:
I think that this will make `recursivePruneCache` unused, so that should probably go as well

Reply:
Removed 👍🏼